reqs/dist.txt

```diff
@@ -1,6 +1,6 @@
 appdirs>=1.3.0
 appnope>=0.1.0; "darwin" in sys_platform
-plover-stroke>=1.0.0dev4
+plover-stroke>=1.0.0
```

Comment:
Mismatch here?

Reply:
No, the minimum supported version is `plover-stroke>=1.0.0`, but we're using the latest release (`1.0.1`) for development / generating the distributions.

Reply:
The API before `1.0.0` is different.